Workbooks/Citrix.json

```diff
@@ -9,283 +9,1741 @@
-        "size": 0,
-        "exportToExcelOptions": "visible",
-        "title": "Breakdown by Risk Indicator",
-        "timeContext": {
-          "durationMs": 604800000
-        },
-        "queryType": 0,
-        "resourceType": "microsoft.operationalinsights/workspaces",
-        "visualization": "piechart"
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 9,
+            "content": {
+              "version": "KqlParameterItem/1.0",
+              "parameters": [
+                {
+                  "id": "78720f79-3173-44cc-856c-e61c42cc0fbf",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "timeRange",
+                  "label": "Select Time Range",
+                  "type": 4,
+                  "value": {
+                    "durationMs": 2592000000
+                  },
+                  "typeSettings": {
+                    "selectableValues": [
+                      {
+                        "durationMs": 3600000
+                      },
+                      {
+                        "durationMs": 43200000
+                      },
+                      {
+                        "durationMs": 86400000
+                      },
+                      {
+                        "durationMs": 604800000
+                      },
+                      {
+                        "durationMs": 2592000000
+                      }
+                    ]
+                  },
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "parameters - 7"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "CitrixAnalytics_userProfile_CL\n| where event_type_s == \"userProfileApp\"\n| summarize sum(cnt_d) by app_s\n| take 10\n| order by sum_cnt_d desc ",
+              "size": 3,
+              "title": "Top 10 User Applications",
+              "timeContext": {
+                "durationMs": 2592000000
+              },
+              "timeContextFromParameter": "timeRange",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "piechart"
+            },
+            "customWidth": "33",
+            "name": "query - 9",
+            "styleSettings": {
+              "showBorder": true
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "CitrixAnalytics_userProfile_CL\n| where event_type_s == \"userProfileDevice\"\n| summarize sum(cnt_d) by device_s\n| take 10\n| order by sum_cnt_d desc ",

```

Comment:
This query is failing. Please look into it.
![image](https://user-images.githubusercontent.com/87809732/140907692-5502fc28-7347-4f17-99a0-2328089e3c38.png)